modules/node/src/cfCore/cfCore.store.ts

```diff
@@ -361,6 +373,14 @@ export class CFCoreStore implements IStoreService {
           .execute();
       }
     });
+    await this.cache.mergeCacheValues(`appInstance:identityHash:${freeBalanceAppInstance.identityHash}`, 60000, {
```

Comment:
This class repeatedly looks up the same data using different criteria. The caching scheme here maps that criteria back to a domain object, and manages purging/merging the cache with new values when the underlying data changes.